app/views/pages/privacy_notice.md

```diff
@@ -49,6 +50,7 @@ title: Register trainee teachers privacy notice
   <li>equality and diversity information</li>
   <li>National Insurance number</li>
```

Comment:
Didn't know we collect NI no

Reply:
that's an interesting point. this text was introduced a [couple of months ago](https://github.com/DFE-Digital/register-trainee-teachers/commit/f99c83d2aa) but afaik we haven't collected Nino for quite some time.